src/test/java/com/oreilly/learningsparkexamples/java/ChapterSixExampleTest.java

```diff
@@ -0,0 +1,187 @@
+package com.oreilly.learningsparkexamples.java;
+
+import static org.mockito.AdditionalMatchers.or;
+import static org.mockito.Matchers.anyBoolean;
+import static org.mockito.Matchers.isA;
+import static org.mockito.Matchers.isNull;
+
+import com.diffblue.deeptestutils.Reflector;
+import com.diffblue.deeptestutils.mock.DTUMemberMatcher;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.oreilly.learningsparkexamples.java.CallLog;
+import com.oreilly.learningsparkexamples.java.ChapterSixExample.SumInts;
+import com.oreilly.learningsparkexamples.java.ChapterSixExample.VerifyCallLogs;
+import com.oreilly.learningsparkexamples.java.ChapterSixExample;
+import org.apache.spark.Accumulator;
+import org.apache.spark.broadcast.Broadcast;
+import org.eclipse.jetty.client.ContentExchange;
+import org.eclipse.jetty.client.HttpClient;
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.Timeout;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import scala.Tuple2;
+
+import java.lang.reflect.Array;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+
+@org.junit.runner.RunWith(org.powermock.modules.junit4.PowerMockRunner.class)
+public class ChapterSixExampleTest {
+
+  @Rule public ExpectedException thrown = ExpectedException.none();
+  @Rule public Timeout globalTimeout = new Timeout(10000);
+
+  @Test
+  public void callInput0Output0() {
+
+    // Arrange
+    final VerifyCallLogs objectUnderTest = new VerifyCallLogs();
+    final CallLog[] input = {};
+
+    // Act
+    final CallLog[] retval = objectUnderTest.call(input);
+
+    // Assert result
+    Assert.assertArrayEquals(new CallLog[] {}, retval);
+  }
+
+  @Test
+  public void callInput0Output02()
```

Comment:
I suggest removing this test which uses reflection. That would then mean the above import statements for reflection can be removed.